packages/trader/src/Modules/Trading/Containers/Multiplier/risk-management-dialog.tsx

```diff
@@ -83,9 +93,9 @@ const RiskManagementDialog = observer(({ is_open, onClose, toggleDialog }) => {
         toggleDialog();
     };
 
-    const resetAndClose = (...args) => {
+    const resetAndClose = () => {
         setState(applied_risk_management_state);
-        onClose(...args);
+        onClose();
```

Comment:
`onClose` function is actually a `() => setDialogOpen(false)` (passed as a props from _risk-management-info.tsx_), so there is no need in `...args`